src/game.cpp

```diff
@@ -1 +1,71 @@
-#include "game.h"
\ No newline at end of file
+#include "game.h"
+
+#define PI 3.14159265f
+#define CCW90 90
+#define DEGREES_IN_A_CIRCLE 360.f
+
+Game::Game()
+{
+    this->LMBHeldDown = false;
+    this->velocity = 0.f;
+    this->angle = 90.f;
+}
+
+void Game::init()
+{
+    this->window.create(sf::VideoMode(DISPLAY_CONSTS::WIDTH, DISPLAY_CONSTS::HEIGHT), "FeelsRocketMan");
+}
+
+void Game::run()
+{
+    while (this->window.isOpen())
+    {
+        moveShip();
+        sf::Event event;
+        while (this->window.pollEvent(event))
+        {
+            if (event.type == sf::Event::Closed)
+            {
+                terminate();
+            }
+            else if (event.type == sf::Event::MouseButtonReleased && event.mouseButton.button == sf::Mouse::Left)
+            {
+                auto [shipx, shipy] = this->ship.getPosition();
+                auto [mousex, mousey] = sf::Mouse::getPosition(this->window);
+                this->velocity = SHIP_CONSTS::THRUST_L1;
+                this->angle = (std::atan2(shipy -  static_cast<float>(mousey), shipx - static_cast<float>(mousex)) * 180.f/(PI));
+                this->LMBHeldDown = false;
+            }
+            else if ((event.type == sf::Event::MouseButtonPressed && event.mouseButton.button == sf::Mouse::Left) || this->LMBHeldDown == true)
+            {
+                auto [shipx, shipy] = this->ship.getPosition();
+                auto [mousex, mousey] = sf::Mouse::getPosition(this->window);
+                this->ship.trackMouse(std::atan2(shipy -  static_cast<float>(mousey), shipx - static_cast<float>(mousex)) * 180.f/(PI) + CCW90);
```

Comment:
seems like this conversion to degrees from radians and vice versa will be common operations that you'll be doing, why not define a macros for it?